### PR TITLE
[IMP] README.md file

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 [![Build Status](https://travis-ci.org/OCA/purchase-workflow.svg?branch=7.0)](https://travis-ci.org/OCA/purchase-workflow)
```
Comment (pedrobaeza) [inline README.md:2]: Please add runbot badge

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 This project aim to deal with modules related to manage purchase and their related workflow. You'll find modules that:
 
  - Allow to easily group purchase order
  - Choose the cheapest supplier
  - Add validation step
- - ...
+
+Translation Status
+----------------
```
Comment (pedrobaeza) [inline README.md]: Put the same length for the underlining

File: README.md (hunk 1 above)
Comment (pedrobaeza) [inline README.md:2]: I'm wondering if we should rename this to "Travis Status" to differentiate it from the runbot one...

Comment (max3903) [inline README.md:2]: Well, it's another discussion. I am compliant with the template:
https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md

Comment (pedrobaeza) [inline README.md:2]: Yeah, I know, it's to know your opinion

Comment (max3903) [inline README.md:2]: :) I agree. Name of the tools would make more sense and be coherent with the other badges, but honestly, this is not even a cosmetic issue as it is an alternative to the image. I won't sleep any better if it changes.

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**